### DISCUSSION TITLE
Autoimport was commented out when migrating from csv-importer. What is it intentional?

### DISCUSSION THREAD

**Discussion by sushi30** [firefly-iii/firefly-iii#5355] [Firefly III] [answered]
CSV importer:
https://github.com/firefly-iii/csv-importer/blob/440deb3058d8dac8fa60f9694038aaf8a2149839/routes/web.php#L42-L44

Data importer:
https://github.com/firefly-iii/data-importer/blob/13379fa302c35037f19c67015da2e4d9513b3aeb/routes/web.php#L104-L106

If not.. Can it be uncommented and included in next release?

**Comment by JC5** [comment]
Yes, and yes.

**Comment by JC5** [comment, marked as answer]
Yes, and yes.
